assets/tsconfig.json

```diff
@@ -0,0 +1,64 @@
+{
+    "compilerOptions": {
+        /* Basic Options */
+        "target": "es5", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+        "module": "commonjs", /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+        "lib": [
+            "dom",
+            "es2017"
+        ], /* Specify library files to be included in the compilation. */
+        "allowJs": true, /* Allow javascript files to be compiled. */
+        // "checkJs": true,                    /* Report errors in .js files. */
+        "jsx": "react", /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
+        // "declaration": true,                /* Generates corresponding '.d.ts' file. */
+        // "declarationMap": true,             /* Generates a sourcemap for each corresponding '.d.ts' file. */
+        // "sourceMap": true,                  /* Generates corresponding '.map' file. */
+        // "outFile": "./",                    /* Concatenate and emit output to single file. */
+        // "outDir": "./",                     /* Redirect output structure to the directory. */
+        // "rootDir": "./",                    /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
+        // "composite": true,                  /* Enable project compilation */
+        // "removeComments": true,             /* Do not emit comments to output. */
+        // "noEmit": true,                     /* Do not emit outputs. */
+        // "importHelpers": true,              /* Import emit helpers from 'tslib'. */
+        // "downlevelIteration": true,         /* Provide full support for iterables in 'for-of', spread, and destructuring when targeting 'ES5' or 'ES3'. */
+        // "isolatedModules": true,            /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */
+        /* Strict Type-Checking Options */
+        // "strict": true, /* Enable all strict type-checking options. */
```

Comment:
Is it practical for us to enable full `strict` mode instead of just the `strictNullChecks`?

Reply:
i can check